Pod/Classes/PINRemoteImageCategoryManager.m

```diff
@@ -172,7 +172,7 @@ + (void)setImageOnView:(id <PINRemoteImageCategory>)view
     if (placeholderImage) {
         [view pin_setPlaceholderWithImage:placeholderImage];
     }
-    PINRemoteImageManagerDownloadOptions options = PINRemoteImageManagerDownloadOptionsNone;
+    PINRemoteImageManagerDownloadOptions options = [view pin_defaultOptions];
```

Comment:
Hey, thanks for getting back to me. I'm not sure I follow -- what subclasses do you mean? This won't break any subclasses of UIImageView/UIButton/FLAnimatedImageView. This is a perfectly valid subclass:

```
@import UIKit;
#import "UIImageView+PINRemoteImage.h"

@interface UIImageViewSubclass : UIImageView
@end
@implementation UIImageViewSubclass

//Not implementing pin_defaultOptions, but I want to override GIF ignoring
- (BOOL)pin_ignoreGIFs {
    return YES;
}

@end
```

I guess it would break any classes outside of this library that conform to the `PINRemoteImageCategory` but I'm not sure if its a good idea to support that?